test/integration_test.go

```diff
@@ -586,13 +546,7 @@ func TestTerragruntSetsAccessLoggingForTfSTateS3BuckeToADifferentBucketWithGiven
 		lockTableName,
 		"remote_terragrunt.hcl",
 	)
-	localBackendTerragruntConfigPath := filepath.Join(examplePath, "local_terragrunt.hcl")
-
-	// Pass 1 with local backend. This should only download and setup the terragrunt-cache. Note that the first pass has
-	// to be an apply so that the state file is created. Otherwise, terragrunt short circuits the problematic routine.
-	runTerragrunt(t, fmt.Sprintf("terragrunt apply -auto-approve --terragrunt-non-interactive --terragrunt-config %s --terragrunt-working-dir %s", localBackendTerragruntConfigPath, examplePath))
```

Comment:
This test case and the one below look like the copy/pasted the logic from `TestTerragruntHonorsS3RemoteStateSkipFlagsRegression` around running first with a local backend and then with an S3 backend, so they hit the same interactive prompt. However, I don't think these two test cases needed this two-step process, so I updated them to _only_ run with the S3 backend, which seems to work.